test/GovernorCountingFractional.t.sol

```diff
@@ -324,7 +324,7 @@ contract GovernorCountingFractionalTest is Test {
     assertEq(abstainVotes, abstainVotesCast);
 
     IGovernor.ProposalState status = IGovernor.ProposalState(uint32(governor.state(_proposalId)));
-    if (forVotes > againstVotes && forVotes >= governor.quorum(block.number)) {
+    if (forVotes > againstVotes && forVotes + abstainVotes >= governor.quorum(block.number)) {
```

Comment:
Nit: With multi-operator clauses like this I really like extra parens for clarity. Personally I would write it as:

```solidity
if ( (forVotes > againstVotes) && ((forVotes + abstainVotes) >= governor.quorum(block.number)) ) {
```

Or at a minimum:

```solidity
if ( forVotes > againstVotes && (forVotes + abstainVotes) >= governor.quorum(block.number) ) {
```

I know this is a subjective preference. Will leave it up to you to decide and self resolve.

Reply:
Sure, I'm good with the second. Fixed in 450b444